docs/maatregelen/security_by_design.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Hanteer security-by-design principes
+toelichting: De principes van ‘security-by-design’ zijn uitgangspunten bij de ontwikkeling van het algoritme.
+vereiste:
+- beveiliging_informatie_en_informatiesystemen
+- automatische_logregistratie
+levenscyclus:
+- ontwerp
+- dataverkenning-en-datapreparatie
+- ontwikkelen
+- verificatie-en-validatie
+- implementatie
+- monitoring-en-beheer
+bouwblok:
+- technische-robuustheid-en-veiligheid
+- governance
+- transparantie
+- menselijke-controle
+rollen:
+- projectleider
+- informatiebeheerder
+- security-officer
+hide:
+- navigation
+---
+
+<!-- tags -->
+
+## Maatregel
+
+Hanteer principes van ‘security-by-design’ (informatiebeveiligingsmaatregelen) als uitgangspunten bij de ontwikkeling van het algoritme.
+Stel vast welke principes horen bij security-by-design en welke relevant zijn voor het ontwerp of de ontwikkeling van het algoritme.
```

Comment:
Moeten wij die principes hier niet gaan aanreiken? Blijft anders wat vaag. Pakken we nu wel de essentie beet van security by design? Volgens mij is dit vooral dat tijdig de ISO wordt betrokken bij de werkzaamheden en dat deze aangeeft wat moet worden gedaan?